src/libs/ajax.js

```diff
@@ -65,9 +83,10 @@ const withRetryOnError = _.curry(wrappedFetch => async (...args) => {
       // requesterPaysError may be set on responses from requests to the GCS API that are wrapped in withRequesterPays.
       // requesterPaysError is true if the request requires a user project for billing the request to. Such errors
       // are not transient and the request should not be retried.
-      const shouldRetry = !error.requesterPaysError
 
+      const shouldRetry = !error.requesterPaysError
       if (!shouldRetry) {
+        captureFailure()
```

Comment:
Should this be outside the `if (!shouldRetry)` block? As is, this would only be called for errors that are not retried (currently, only requester pays errors from the GCS API).